libraries/botbuilder/src/streaming/streamingHttpClient.ts

```diff
@@ -34,8 +34,12 @@ export class StreamingHttpClient implements HttpClient {
      */
     public async sendRequest(httpRequest: WebResource): Promise<HttpOperationResponse> {
         if (!httpRequest) {
-            throw new Error('SendRequest invalid parameter: httpRequest should be provided');
+            throw new Error('StreamingHttpClient.sendRequest(): missing "httpRequest" parameter');

```

Comment:
Changing error message strings in JS can have the effect of being a breaking change due to JS devs doing regex matches for handling error cases. This is probably a minuscule amount of risk to change in this case, though.

Reply:
That's true, I think in this case I think we can get away with it. But if there is major push-back I can   take this out...